api/src/pcapi/templates/invoices/invoice.html

```diff
@@ -244,7 +244,7 @@ <h3>
         <td>{{ total_used_bookings_amount|fr_currency(is_caledonian_invoice) }} {{ currency_symbol }}</td>
         <td></td>
         <td>{{ total_contribution_amount|fr_currency(is_caledonian_invoice) }} {{ currency_symbol }}</td>
-        <td>{{ total_reimbursed_amount|fr_currency_opposite(is_caledonian_invoice) }} {{ currency_symbol }}</td>
+        <td>{{ total_reimbursed_amount|fr_currency_opposite(is_caledonian_invoice) }} {{ currency_symbol }}{{ ' (' + total_reimbursed_amount|fr_currency_opposite(not is_caledonian_invoice) +' €)' if is_caledonian_invoice else '' }}</td>
```

Comment:
y'a moyen de simplifier un peu : 

```suggestion
        <td>{{ total_reimbursed_amount|fr_currency_opposite(is_caledonian_invoice) }} {{ currency_symbol }}{% if is_caledonian_invoice %} ({{ total_reimbursed_amount|fr_currency_opposite }}&nbsp;€){% endif %}</td>
```